docs/src/_guide/advanced-topics.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Advanced Topics
+description: Learn about advanced topics in the Meltano.
+layout: doc
+weight: 10
+---
+
+## Installing Optional Components
+
+Most of Meltano's features are available without installing any additional packages. However, some niche or environment-specific features require the installation of [extras](https://peps.python.org/pep-0508/#extras). The following extras are currently supported:
```

Comment:
Similar comment to above, adding a comment about this being a Python, not a Meltano, extra.